compose/compose.go

```diff
@@ -226,7 +240,7 @@ func getZero(idx int, raft string) service {
 	basePort := zeroBasePort + opts.PortOffset
 	grpcPort := basePort + getOffset(idx)
 
-	svc := initService(basename, idx, grpcPort)
+	svc := initService(basename, idx, grpcPort, opts.ContainerAlias)
```

Comment:
Probably you don't need to pass in the opts.ContainerAlias. `opts` is global, so you can directly use `opts.ContainerAlias` in this function.